burndown_chart.py

```diff
@@ -96,25 +96,33 @@ def init_actual_remaining(sprint_info):
             actual_remaining[str_date] = {'value': 0, 'story_list': []}
     return actual_remaining
 
+def is_open(issue):
+    return issue.status.name != "Resolved" and issue.status.name != "Rejected"
 
 def query_redmine(sprint_info):
     project_list = ['suseqa', 'openqav3', 'openqatests']
     str_start_date = sprint_info['start_date'].strftime(dt_format)
     str_due_date = sprint_info['due_date'].strftime(dt_format)
     date_interval = '><' + str_start_date + '|' + str_due_date
     rm = Redmine('https://progress.opensuse.org', key='XXXXXXX')
-    issues = rm.issue.filter(project_ids=project_list, status_id='*', due_date=date_interval, subject="~[" + team + "]")
+    issues = rm.issue.filter( project_ids=project_list,
+                              status_id='*',
+                              due_date=date_interval,
+                              subject="~[" + team + "]" )
     print("Sprint start date: " + str(sprint_info['start_date']))
     print("Sprint end date:   " + str(sprint_info['due_date']))
 
     # Exclude epics
     issues = [issue for issue in issues if "[epic]" not in issue.subject]
     # exclude sagas
     issues = [issue for issue in issues if "[saga]" not in issue.subject]
```

Comment:
We could use `any` here for those two filters. Create a list of exclusion and apply it. Let me check how could be the syntax.

Reply:
Not sure if any fits here as it returns true or false depending on the list. But I guess we can shorten lambda

Reply:
I believe it would be something like this:
```exclude_list = ['[epic]', '[saga]']
    issues = [issue for issue in issues if all(x not in issue.subject for x in exclude_list)]
```

Reply:
this is terribly hard to read, I will just combine conditions in a single if statement

Reply:
should be better now, please, take a look

Reply:
yes, for this case it is much more simple in that way. I just get excited about those any() and all() very handy features that I didn't know before.